paimon-core/src/main/java/org/apache/paimon/operation/OrphanFilesClean.java

```diff
@@ -102,6 +94,7 @@ public OrphanFilesClean(FileStoreTable table) {
         this.fileIO = table.fileIO();
         this.location = table.location();
         this.partitionKeysNum = table.partitionKeys().size();
+        this.schemaManager = table.schemaManager();
```

Comment:
No need to change the visibility of this method. Just create a new schema manager.

Reply:
done